src/components/Recipes/Recipes.jsx

```diff
@@ -173,13 +182,17 @@ export default function Recipes(props) {
 
   return (
     <div className="recipes">
-      <h2 className="recipes-heading">Recipes</h2>
+      <div className="recipes-hero">
+        <img className="utensil-img" id="left" src={ leftUtensil } alt="fork knife" />
+        <h2 className="recipes-heading">Recipes Just For You</h2>
+        <img className="utensil-img" id="right" src={ rightUtensil } alt="fork knife" />
+      </div>
```

Comment:
Could these icons be added as part of the CSS of the `recipes-hero` instead of as html <img> tags?